popup/popup.css

```diff
@@ -264,9 +266,38 @@ html[style*="border"] .entry:nth-child(11):before {
 }
 
 .entry .actions > * {
-  display: inline-block;
-  padding: 0 1px;
-  margin: 0 1px;
+  height: 26px;
+  width: 18px;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.entry .actions > .menu-button {
+  width: 14px;
```

Comment:
I think we should normalize these icons so we don't have to set different width/height individually.

Reply:
That sounds good until you screw around with editing SVG for an hour or so, and you've got icons that are all the "same size" which don't look anywhere near as good as what you started with.

I swapped out the kinda funky looking external icons and reused the pixel perfect version tophf made for the manager. It's 20px, but the actual icon is 13px centered in the middle. The cog is 14px x 14px, edit is 14px x 16px (naturally), and the menu icon is naturally narrow.

Container buttons range from 18px to 22px (with widths and/or padding), which is all based on spacing of icons while still leaving adequate clickable areas with no margins. Making them all the same exact widths would be a nightmare to achieve even spacing. 

The menu button is narrow because the icon is narrow. I eyeballed the spacing while leaving what I think is a big enough clickable area. I'm eyeballing all the action buttons and their icons, that's kinda the nature of using SVG in my experience.

I'm pretty happy with them currently, but if the couple pixels here or there bug you, I wish you luck trying to improve them. I'm not messing with SVG icons I think look good.